inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/RecommenderEditorPanel.java

```diff
@@ -511,4 +511,9 @@ public void validate(IValidatable<String> aValidatable)
             }
         }
     }
+    
+    public IModel<Recommender> getRecommenderModel()
```

Comment:
Can be removed. See comment above about passing the model in the constructor.

Reply:
Done.